modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1631,6 +1631,10 @@
 			<description name="Replaces with `compileInclude` the configuration attribute for dependencies in `build.gradle` that are listed at `Include-Resource` property at `bnd.bnd` associated file." />
 			<weight>2</weight>
 		</check>
+		<check name="UpgradeJavaAddAddressMethodCheck">
+			<category name="Upgrade" />
+			<description name="Fill the new parameter of the method 'addAddress' of AddressLocalServiceUtil, AddressServiceUtil, AddressLocalService and AddressService" />
```

Comment:
@TamyrisTorres, can you sort alphabetically?

AddressLocalService, AddressLocalServiceUtil, AddressService, AddressServiceUtil